src/Table/elements.js

```diff
@@ -0,0 +1,171 @@
+import styled, { css } from 'styled-components';
+
+export const Container = styled.div`
+  display: flex;
+  flex-direction: column;
+
+  width: ${({ width }) => width};
+  ${({ height }) => height && css` height ${height}`};
+
+  box-shadow: 0 1px 0 0 hsla(0, 0%, 0%, 0.06);
+`;
+
+export const Header = styled.div`
+  display: flex;
+  flex-direction: column;
+
+  width: 100%;
+`;
+
+export const Body = styled.div`
+  display: flex;
+  flex-direction: column;
+
+  width: 100%;
+`;
+
+const getCellFlex = width => (width === parseInt(width).toString() ? width : 'initial');
+const getCellWidth = width => (width !== parseInt(width).toString() ? width : 'initial');
```

Comment:
Same here, but reversed. You set CSS `width` attribute to `initial` if width of the column is a weight, otherwise the given value (e.g. `10px`, `2rem`).